metafacture-flux/src/main/java/org/metafacture/flux/HelpPrinter.java

```diff
@@ -171,4 +193,19 @@ private static Collection<String> getAvailableArguments(final Class<?> moduleCla
         return Collections.emptyList();
     }
 
+    @SuppressWarnings("IllegalType")
+    private static HashMap<String, List<String>> loadExamples(final String fname) throws IOException {
+        final File f = new File(fname);
+        if (Files.exists(f.toPath())) {
+            final BufferedReader bufferedReader = new BufferedReader(new FileReader(f));
+            final StringBuffer stringBuffer = new StringBuffer();
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                final List<String> tsv = Arrays.asList(line.split("\t"));
```

Comment:
I don't see the necessity to convert the array into a list, but it's not a big issue...

Reply:
Uses arrays now